sample/sampleDurableLogger/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Loggly;
+using Loggly.Config;
+using Serilog;
+using Serilog.Core.Enrichers;
+using Serilog.Enrichers;
+using Serilog.Sinks.RollingFileAlternate;
+
+namespace sampleDurableLogger
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            SetupLogglyConfiguration();
+            var logger = CreateLoggerFactory(@"c:\test\").CreateLogger();
+
+            logger.Information((Exception)null, "test message - app started");
+            logger.Warning((Exception)null, "test message with {@data}", new { p1 = "sample", p2=DateTime.Now });
+            logger.Warning((Exception)null, "test2 message with {@data}", new { p1 = "sample2", p2 = 10 });
+
+            Console.WriteLine("disconnect to test offline. Two messages will be sent. Press enter to send and wait a minute or so before reconnecting or use breakpoints to see that send fails.");
+            Console.ReadLine();
+
+            logger.Information((Exception)null, "second test message - app started");
+            logger.Warning((Exception)null, "second test message with {@data}", new { p1 = "sample2", p2 = DateTime.Now });
+
+
+            Console.WriteLine("Offline messages written. Once you have confirmed that messages have been written locally, reconnect to see messages go out. Press Enter  for more messages to be written.");
+            Console.ReadLine();
+
+            logger.Information((Exception)null, "third test message - app started");
+            logger.Warning((Exception)null, "third test message with {@data}", new { p1 = "sample3", p2 = DateTime.Now });
+
+            Console.WriteLine("back online messages written. Check loggly and files for data. wait a minute or so before reconnecting. Press Enter to temrinate");
+            Console.ReadLine();
+        }
+
+        public static LoggerConfiguration CreateLoggerFactory(string logFilePath)
```

Comment:
I'll change it to CreateLogger and return an ILogger directly. This was a copy / paste mistake from my implementation case-